jipdate.py

```diff
@@ -498,6 +498,20 @@ def get_header():
 
     return ("\n".join(yml_iter) + "\n\n") if yml_iter is not None else "\n"
 
+def get_editor():
+    global g_yml_config
+
+    try:
+        yml_iter = g_yml_config['text-editor']
+    except:
+        # Probably no "text-editor" section in the yml-file.
+        return True
+
+    if yml_iter == "no" or yml_iter == "NO" or yml_iter == "No":
```

Comment:
Do we really need to add : 
"use-text-editor: True"
in default config.yml file as the default value when not present is True ?